randovania/games/am2r/presets/starter_preset.rdvpreset

```diff
@@ -136,7 +136,13 @@
             "mode": "vanilla",
             "types_state": {
                 "door": {
-                    "can_change_from": [],
+                    "can_change_from":
+                    [
+                        "Normal Door",
+                        "Missile Door",
+                        "Power Bomb Door",
+                        "Super Missile Door"
+                    ],
                     "can_change_to": []
```

Comment:
you should at least include the unlocked weakness in the default change_to settings - otherwise it'll be unchecked in the UI and greyed out, but it'll always still be an option regardless. more generally, it's conventional to enable all the vanilla door types in the default preset, and disable any custom types. so long as the actual mode is set to vanilla, enabling these doors will have no effect on generation